client/state/data-layer/wpcom/posts/revisions/index.js

```diff
@@ -64,8 +74,17 @@ export const receiveError = ( { dispatch }, { siteId, postId }, next, rawError )
  * @param {Array} revisions raw data from post revisions API
  */
 export const receiveSuccess = ( { dispatch }, { siteId, postId }, next, revisions ) => {
+	const normalizedRevisions = map( revisions, normalizeRevision );
+
+	forEach( normalizedRevisions, ( revision, index ) => {
+		revision.changes = index === normalizedRevisions.length - 1
+			? { added: 0, removed: 0 }
+			: countDiffWords( diffWords( normalizedRevisions[ index + 1 ].content, revision.content ) );
+	} );
+
 	dispatch( receivePostRevisionsSuccess( siteId, postId ) );
-	dispatch( receivePostRevisions( siteId, postId, map( revisions, normalizeRevision ) ) );
+	dispatch( receivePostRevisions( siteId, postId, normalizedRevisions ) );
+	dispatch( requestUsers( siteId, uniq( map( normalizedRevisions, 'author' ) ) ) );
```

Comment:
The initial reason for it to be in there was to bundle revisions and users by dispatching `receivePostRevisions`, `receivePostRevisionsSuccess` and `receiveUsers` in the same tick. But I dropped it, from a UX pov, having the author names is nice but not as important as accessing the revisions asap: it's better to have a first render of the revisions while we request more information about the various authors imho.

I've updated this PR to remove this and will probably bundle `QueryUsers` with its "usage" in the last PR for revisions.